frontend/src/app/edit-claim/edit-claim.component.ts

```diff
@@ -47,22 +47,35 @@ export class EditClaimComponent implements OnInit {
     }
   }
 
-  deleteFile(filename: string): void {
-    // FIXME: This is not correct
-    this.service
-      .deleteFile(filename, this.vin)
-      .subscribe(
-        undefined,
-        error => console.log(error),
-        () => alert('FILE DELETED'),
-      );
-    // TODO: show visual confirmation of deletion
+  archiveFile(filename: string): void {
+    this.service.archiveFile(filename, this.vin).subscribe(
+      undefined,
+      error => console.log(error),
+      () => {
```

Comment:
This usage of subscribe is what is being deprecated. This will need to be replaced next - with the refactor of the frontend I will be adding more async pipes to unwrap data so this will be unneeded.